documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -0,0 +1,79 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'prometheus',
+        rules: [
+          {
+            alert: 'PromBadConfig',
+            expr: |||
+              prometheus_config_last_reload_successful{%(prometheusSelector)s} == 0
+            ||| % $._config,
+            'for': '15m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              mesage: 'Prometheus failed to reload config, see container logs',
+            },
+          },
+          {
+            alert: 'PromAlertmanagerBadConfig',
```

Comment:
This should be in the Alertmanager repo, no?

Reply:
I think you mentioned @mxinden was working on a mixin for Alertmanager - so I wait for that or JFDI?

Reply:
This is gone, so I resolve this conversation.